includes/collection/class-following.php

```diff
@@ -133,6 +136,14 @@ public static function unfollow( $post, $user_id ) {
 						'key'   => '_activitypub_object_id',
 						'value' => $post->guid,
 					),
+					array(
+						'key'   => '_activitypub_activity_type',
+						'value' => 'Follow',
+					),
+					array(
+						'key'   => '_activitypub_activity_actor',
+						'value' => $actor_type,
```

Comment:
@pfefferle What's the benefit of adding actor type? Would we expect that to ever be different from `author`?

Reply:
For blog actor and application actor the user id is `0` so we need the actor type to address the "virtual" users properly.